web/packages/teleterm/src/sharedProcess/api/protogen/ptyHostService_grpc_pb.d.ts

```diff
@@ -1,4 +1,4 @@
-// package: 
```

Comment:
This is autogenerated code. Is this diff doing to show up every time we re-generate the protobufs?

If so, can we ignore the linter on these files?

Reply:
I would just revert these changes then. Otherwise the next person to regenerate protobufs will wonder if they did somethign wrong when they see a diff.

Reply:
done!